pkg/controller/appdefinition/volume.go

```diff
@@ -200,7 +200,8 @@ func toPVCs(req router.Request, appInstance *v1.AppInstance) (result []kclient.O
 			pvc.Spec.VolumeName = pvName
 
 			if volumeRequest.Size == "" {
-				pvc.Spec.Resources.Requests[corev1.ResourceStorage] = *v1.DefaultSize
+				// TODO - should we do some sort of nil pointer validation here?
+				pvc.Spec.Resources.Requests[corev1.ResourceStorage] = *appInstance.Status.Defaults.VolumeSize
```

Comment:
Want to put a special highlight out for this change, thoughts?

Reply:
Like in the changelog? I'm not sure it's something that users would understand.

Reply:
I meant that I want to talk about this line of code in GitHub. The code has actually changed since I wrote this comment since I had to update it to fix some tests. I'll resolve this and open another comment.